### PR TITLE
add usage docs

### PR DESCRIPTION
**Pull request by RazcoDev**

### CODE REVIEW

File: docs/manage-your-account/workspace-usage.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+To view your workspace usage, go to the **Projects** screen on the Sidebar.
+The workspace usage is displayed in the top right corner of the screen.
+In addition to the current usage, clicking on the (?) icon will show you the usage history of your workspace tenants and MAUs.
+
+<video controls autoPlay loop muted playsInline>
+  <source src={orgUsage} type="video/mp4" />
+</video>
+
+The **Workspace Usage** screen gives you a comprehensive overview of your workspace's usage. It shows you the number of:
```
Comment (filipermit) [inline docs/manage-your-account/workspace-usage.mdx:22]: I think there is no need to mention "Workspace usage" so many times. Let's just say:
This section gives you a comprehensive overview of your current usage. It shows you the number of:"

File: docs/manage-your-account/workspace-usage.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Keeping track of your workspace usage is essential to ensure that your team is operating at peak efficiency and that your
+organization is getting the most out of Permit.
+
+## Workspace Usage
+
+To view your workspace usage, go to the **Projects** screen on the Sidebar.
+The workspace usage is displayed in the top right corner of the screen.
+In addition to the current usage, clicking on the (?) icon will show you the usage history of your workspace tenants and MAUs.
+
+<video controls autoPlay loop muted playsInline>
```
Comment (filipermit) [inline docs/manage-your-account/workspace-usage.mdx:18]: One small thing with the recordings: 
- I want to make sure we have a consistent way out UI recordings look (and this is something I need to change too) but lets remove all padding and rounded corners from screen.studio settings. ;)

Comment (RazcoDev) [reply docs/manage-your-account/workspace-usage.mdx:18]: Done :)

File: docs/manage-your-account/workspace-usage.mdx (hunk 1 above)
Comment (RazcoDev) [reply docs/manage-your-account/workspace-usage.mdx:22]: Done, also changed the title

**Comment by filipermit** [comment]
Nice work! 👍 👍

**Review by filipermit — APPROVED** [review]

**Pull request merged**